src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -1029,7 +1010,7 @@ void GLHEVert::calcGFunctions(EnergyPlusData &state)
 void GLHEVert::setupTimeVectors()
 {
 
-    constexpr int numDaysInYear(365);
+    constexpr int numDaysInYear = 365;
```

Comment:
This is not already defined somewhere?

Reply:
I see this one, state.dataWeatherManager->NumDaysInYear